cargo-shuttle/src/lib.rs

```diff
@@ -657,6 +657,7 @@ impl Shuttle {
             .stop(tonic::Request::new(stop_request))
             .or_else(|err| async {
                 runtime.kill().await?;
+                error!(status = ?err, "killed the runtime by force because stopping it errored out");
```

Comment:
What if we do `trace!` here, then the user won't see an error (which won't be relevant to them in the event of `CTRL C` since as far as they are concerned it was a success, not an error, right?)?